glue/configure_includes/configure_linux_arm.mk

```diff
@@ -27,36 +27,54 @@ else
   TEMP_TARGET_DATASIZE:=32
 endif
 
+ifeq ($(OPENJ9_DOCKER_CC_ARM),1)
+  TMP_HOST = $$(OPENJ9_CC_PREFIX)
```

Comment:
Can you change the name of the environment variable to remove the OpenJ9 from it?  The OMR project is / can be used for many downstream projects so we should not be adding any new OpenJ9 specific options.  I am assuming this would be useful for other projects as well if it had a different name.  Thanks.

Reply:
Hi @charliegracie ,
In this case OMR is the consumer of the variable which is defined in the Docker file added to openJ9 in https://github.com/eclipse/openj9/pull/622. The variable is named for consistency with other variables defined in that file. OPENJ9_DOCKER_CC_ARM specifically is destined to be removed as soon as resources permit, as it is something of a quick fix to allow rapid prototyping of builds for armhf.

Reply:
If I understand the interaction between OpenJ9 and OMR correctly, openj9/runtime/gc_glue_java/configure_includes/configure_linux_arm.mk is the actualmk used by openj9 and the omr/glue is a set of examples to work from.  If this is the case, then these changes need to happen in the openj9 repository.  Can you confirm or deny that @charliegracie?

Reply:
Ah, that's interesting. Changes to openj9/runtime/gc_glue_java/configure_includes/configure_linux_arm.mk were contributed in https://github.com/eclipse/openj9/pull/686.
If the files changed in this PR are not actually needed for building OpenJDK then we can scrap them and avoid the whole issue of what to call the env vars.

Reply:
Something in this PR is needed as the build fails if I revert to the original branch.

Reply:
It's the change to omrmakefiles/rules.linux.mk that is needed in order to get the -marm option to the new compiler, so I can reduce this PR to just that change.